applications/alumniprofile/models.py

```diff
@@ -159,9 +169,10 @@ class PastExperience(models.Model):
     start_date = models.DateField()
     end_date = models.DateField(blank=True, null=True)
 
+
 class Education(models.Model):
     profile = models.ForeignKey(Profile, on_delete=models.CASCADE)
-    degree = models.CharField(max_length=150)
+    degree = models.ForeignKey(Degree, on_delete=models.CASCADE)
```

Comment:
It should be `models.SET_NULL` instead. If we delete a degree, it should not delete the `Education` instance I guess.